mirrormanager2/app.py

```diff
@@ -373,14 +375,22 @@ def host_new(site_id):
 
         try:
             SESSION.flush()
+            status = 'Host added'
             flask.flash('Host added')
         except SQLAlchemyError as err:
             SESSION.rollback()
+            status = 'Could not create the new host'
```

Comment:
As it is written, it looks like yes.  But I guess I'd advise against it in this case.